draft-irtf-cfrg-hpke.md

```diff
@@ -157,7 +178,10 @@ proofs of IND-CCA2 security, or fail to provide test vectors.
 This document defines an HPKE scheme that provides a subset
 of the functions provided by the collection of schemes above, but
 specified with sufficient clarity that they can be interoperably
-implemented and formally verified.
+implemented and formally verified. It is secure against (adaptive)
+chosen ciphertext attacks (IND-CCA2 secure) under standard assumptions
+about the underlying primitives {{HPKEAnalysis}}. A summary of this
```

Comment:
Will we make cryptographers nervous when we say “standard assumptions” but use the ROM? Assuming ROM for HPKE or Hash is pretty standard, but it is not the “standard _model_”. We don't say the word _model_ here, so it might be ok… Is “standard” here meant in the sense of “common”, or in the sense of “classical” (not post-quantum)? Maybe we could use one of these two instead of standard.

Reply:
Hmm... I hadn't considered that. I replaced this with classical, as that seems to best describe what we've done.